recipes/xtensor/run_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
This actually works. Just that

 - the centos image does not have a recent enough gcc.
 - the OSX image does not have recent enough clang.